cogs/general.py

```diff
@@ -660,8 +662,15 @@ async def dog(self, ctx):
 		imagepath = os.path.join(cat_dir, random.choice(os.listdir(cat_dir)))
 		await ctx.send(file=discord.File(imagepath))
 
-
+	@commands.command(aliases = ["rss"])
+	async def blog(self,ctx):
+		""" Pulls the newest blog post for Dota 2"""
+		feed = await get_html(r'https://blog.dota2.com/feed')
+		blog = feedparser.parse(feed)
+		title = "Dota 2 Blog"
+		embed = rsstools.create_embed(title, blog.entries[0])
+		await ctx.send(embed = embed)
```

Comment:
I know when asked I originally said the command should go in general.py, but I think it actually probably makes a lot more sense to put it in one of the dota cogs. Could you move it to dotabase.py?